Packs/AzureActiveDirectory/ReleaseNotes/1_3_16.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Azure Active Directory Identity Protection  (Deprecated)
+
+- Fixed an issue where changes made to the *Authorization code* parameter were not being reflected in the integration code, resulting in the continued use of the first parameter.
```

Comment:
```suggestion
Fixed an issue where changes made to the *Authorization code* parameter were not being reflected in the integration code, resulting in the continued use of the first parameter.
```
Remove the bullet from each of the RNs.  There is no need for a bullet when there is only one item.